trade-finance-manager-ui/server/controllers/facilities/index.js

```diff
@@ -3,7 +3,8 @@ const CONSTANTS = require('../../constants');
 
 const getFacilities = async (req, res) => {
   const { userToken } = req.session;
-  const apiResponse = await api.getFacilities();
+  // queryParams here?
```

Comment:
Interesting that they never passed some parameters through here before - I'd leave a comment saying query parameters have been implemented in APIs but aren't used in the frontends currently